luarules/gadgets/mo_ffa.lua

```diff
@@ -45,94 +45,16 @@ if gadgetHandler:IsSyncedCode() then
 		if Spring.GetGameFrame() >= leaveWreckageFromFrame then
 			GG.wipeoutWithWreckage = leaveWreckage
 		end
-	end
-
```

Comment:
LOTS of changes to this file. Can we make sure this is all related? Some of it looks extraneous.